lib/components/narrative/metro/metro-error-renderer.tsx

```diff
@@ -21,7 +21,7 @@ const Container = styled.li`
   grid-template-rows: 1fr max-content;
   list-style-type: none;
   margin: 0;
-  padding: 0 1em;
+  padding: 1em;
```

Comment:
Why was this change made? It's having an impact outside the nearby view

Reply:
It was dependent on the header existing to make the component look right. With no header the icon is the tallest element, and with no padding it just went right to the edge. I'd rather have the padding on the wrapper component instead of on the children.

Reply:
I'm not sure I like that we're changing unrelated styles. Could we extract the shared styles and then maintain the current nearby view display? Same goes for the support for an error with no header

Reply:
found another way to get the padding I needed